openstudiocore/src/model/CentralHeatPumpSystemModule.cpp

```diff
@@ -99,19 +99,17 @@ namespace detail {
 
   /** This method clones the CentralHeatPumpSystemModyle but sets the chillerHeaterModulesPerformanceComponent to the same as the original
    * By using the "children" method and listing the chillerHeaterModulesPerformanceComponent there ModelObject_Impl::clone will automatically do
-   * the right thing */
+   * the right thing -> NO IT DOESN'T */
   ModelObject CentralHeatPumpSystemModule_Impl::clone(Model model) const
   {
     CentralHeatPumpSystemModule newCentralHPMod = ModelObject_Impl::clone(model).cast<CentralHeatPumpSystemModule>();
-    // ASSERT that it does have the same Perf component as the original Module
-    OS_ASSERT(newCentralHPMod.chillerHeaterModulesPerformanceComponent() == this->chillerHeaterModulesPerformanceComponent());
 
     // If not using "children", then expliclity do it:
-    // bool ok = true;
+    bool ok = true;
     // We don't want to clone the perf object, set it to the same ChillHeaterPerformanceElectricEIR
-    //ok = newCentralHPMod.setChillerHeaterModulesPerformanceComponent(this->chillerHeaterModulesPerformanceComponent());
+    ok = newCentralHPMod.setChillerHeaterModulesPerformanceComponent(this->chillerHeaterModulesPerformanceComponent());
```

Comment:
Assertion fails
```
[ RUN      ] ModelFixture.CentralHeatPumpSystem_Clone
[BOOST_ASSERT] <2> Assertion ok failed on line 112 of virtual openstudio::model::ModelObject openstudio::model::detail::CentralHeatPumpSystemModule_Impl::clone(openstudio::model::Model) const in file /Users/julien/Software/Others/OpenStudio/openstudiocore/src/model/CentralHeatPumpSystemModule.cpp.
Assertion failed: (false), function assertion_failed, file /Users/julien/Software/Others/OpenStudio/openstudiocore/src/model/test/../../utilities/core/Assert.hpp, line 60.```

Reply:
This happens on line 316 of GTest, when I clone into another model... obviously! It's trying to set a chillerHeater that wasn't clone in model2.